sys/install.sh

```diff
@@ -32,6 +32,11 @@ while : ; do
 		shift
 		continue
 		;;
+	"--without-pull")
+		export WITHOUT_PULL=1
+		shift
```

Comment:
```suggestion
```

Reply:
why? the original code shifts it away. Do you want to pass it on thru ARGS?

Reply:
Uhm maybe i missread the patch :? Ill check after lunch